### PR TITLE
BUcash 1.5.0.1 release notes

### PR DESCRIPTION
**Pull request by sickpig**

### CODE REVIEW

File: doc/release-notes-bucash.md [hunk 1]
```diff
@@ -12,11 +12,19 @@ Please report bugs using the issue tracker at github:
...
 with the Bitcoin Cash specifications you could find here:
 
-
 - https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/uahf-technical-spec.md (Aug 1st '17 Protocol Upgrade, bucash 1.1.0.0)
 - https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/nov-13-hardfork-spec.md (Nov 13th '17 Protocol Upgrade, bucash 1.1.2.0)
 - https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/may-2018-hardfork.md (May 15th '18 Protocol Upgrade, bucash 1.3.0.0, 1.3.0.1, 1.4.0.0)
-- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18 Protocol Upgrade, bucash 1.5.0.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18 Protocol Upgrade, bucash 1.5.0.0, 1.5.0.1)
+
+This release will also implements also a set of consensus changes proposed by an alternative implementation, Bitcoin SV,
```
Comment (ptschip) [inline doc/release-notes-bucash.md]: -This release will also implements also a
+This release will also implement a

Comment (sickpig) [reply doc/release-notes-bucash.md]: thanks will fix

**Review by ptschip — APPROVED** [review]

File: doc/release-notes-bucash.md [hunk 2]
```diff
@@ -12,11 +12,19 @@ Please report bugs using the issue tracker at github:
...
 - https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/nov-13-hardfork-spec.md (Nov 13th '17 Protocol Upgrade, bucash 1.1.2.0)
 - https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/may-2018-hardfork.md (May 15th '18 Protocol Upgrade, bucash 1.3.0.0, 1.3.0.1, 1.4.0.0)
-- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18 Protocol Upgrade, bucash 1.5.0.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18 Protocol Upgrade, bucash 1.5.0.0, 1.5.0.1)
+
+This release will also implement a set of consensus changes proposed by an alternative implementation, Bitcoin SV,
+see [SV release notes for ver 0.1.0](https://github.com/bitcoin-sv/bitcoin-sv/blob/master/doc/release-notes.md) for more details.
+Such set of features is **disabled by default**, the default policy is to activate the set of changes as defined by the bitcoincash.org
+[specification](https://github.com/bitcoin-sv/bitcoin-sv/blob/master/doc/release-notes.md).
```
Comment (jonvaage) [inline doc/release-notes-bucash.md]: The URLs on line 21 & 23 are the same. Is this correct? Seems like the latter should direct to a bitcoincash.org page.

Comment (sickpig) [reply doc/release-notes-bucash.md]: you are right indeed. fixing.

**Pull request merged**